packages/taquito-local-forging/src/constants.ts

```diff
@@ -25,8 +25,12 @@ export enum CODEC {
   INT16 = 'int16',
   PARAMETERS = 'parameters',
   ADDRESS = 'address',
+  SMART_ROLLUP_ADDRESS = 'smart_rollup_address',
   SMART_CONTRACT_ADDRESS = 'smart_contract_address',
+  SMART_ROLLUP_CONTRACT_ADDRESS = 'smart_rollup_contract_address',
   VALUE = 'value',
+  PADDED_BYTES = 'padded_bytes',
+  PREPADDED_BYTES_ARRAY = 'prepadded_bytes_array',
```

Comment:
what's the distinction between this and PADDED_BYTES?

Reply:
normal padded bytes are bytes prefixed with bytes length on 4 bytes
array is padded Array < padded array item < padded item > >